doc/user_manual_en.md

```diff
@@ -0,0 +1,111 @@
+# qTox User Manual
+## Index
+
+* [Settings](#Settings)
+* [Groupchats](#Groupchats)
+* [Multi Window Mode](#multi-window-mode)
+* [Keyboard Shortcuts](#keyboard-shortcuts)
+
+## Settings
+### General
+
+#### General Settings
+* _Language:_ Changes which language the qTox interface uses.
+* _Autostart:_ If set, qTox will start when you login on your computer.
+* _Light icon:_ If set, qTox will use a different icon, which is easier to read on black backgrounds.
+* _Show system tray icon:_ If set, qTox will show its icon in your system tray.
+    * _Start in tray:_ On start, qTox will only show its tray icon and no window.
+    * _Minimize to tray:_ The minimize button on the top right, will minimize qTox to its tray icon. There won't be a taskbar item.
+
+* _Auto away after (0 to disable):_ After the specified amount of time, qTox will set your Status to "Away". A setting of 0 will never change your status.
+* _Default directory to save files:_ You can set the directory where qTox puts files you recieved here.
+* _Autoaccept files:_ If set, qTox will automatically accept filetransfers and put them in the directory specified above.
+
+#### Chat
+* _Play sound:_ If checked, qTox will play a sound when you get a new message.
+* _Open window:_ If checked, the qTox window will be opened. <!-- TODO: Will a NEW window be opened? What happens in Multi Window mode? -->
+    * _Focus window:_ If checked, the qTox window will additionally be focused.
+* _Show contacts' status changes:_ ???<!-- TODO: what does this even do?-->
+* _Group chats always notify:_ ???<!-- TODO: what does this even do?-->
+* _Place groupchats at top of friend list:_ If checked, your groupchats will be at the top of the contacts list. <!-- TODO: else what?-->
+* _Faux offline messaging:_ If enabled, qTox will attempt to send messages when a currently offline contact comes online again.<!-- TODO:Is this even correct?-->
+* _Compact contact list:_ If set, qTox will use a contact list layout which takes up less screen space.
+* _Multiple windows mode:_ If enabled, the qTox user interface will be split into multiple independent windows. For details see TODO.<!-- TODO:write detailed explainaiton of multi window mode-->
+    * _Open each chat in an individual window:_ If checked, a new window will be opened for every chat you open.
```

Comment:
>a new window will be opened for every chat you open.

Except for the cases where user manually grouped some contacts together in a single window - in that case a new window won't be opened, but the window in which contact is grouped will be brought (& highlighted) from background.